canonical-web-and-design/ubuntu.com
# Cube shop listing

## Done

- Replace `cube.yaml` content with data from UA contracts API.
- Populate modules list with `/cube/microcerts.json`
- No actions yet, it'll make the PR too big and there are other issues opened for that.

## QA

- Check https://ubuntu-com-10421.demos.haus/cube/microcerts to see a new table with content
- Navigate to /cube/microcerts as a guest user, and make sure it redirects to USSO

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/256

## Screenshots

![advantage-data](https://user-images.githubusercontent.com/14939793/134044050-b8e166e4-d050-4c88-b284-3c5a8d7463e4.png)
